go/v1beta1/storage/grafeas.go

```diff
@@ -0,0 +1,75 @@
+package storage
+
+import (
+	"fmt"
+	grafeasConfig "github.com/grafeas/grafeas/go/config"
+	"github.com/grafeas/grafeas/go/v1beta1/storage"
+	"github.com/liatrio/grafeas-elasticsearch/go/config"
+	"go.uber.org/zap"
+	"net/http"
+)
+
+type newElasticsearchStorage func(*config.ElasticsearchConfig) (*ElasticsearchStorage, error)
+
+type registerStorageTypeProvider func(string, *grafeasConfig.StorageConfiguration) (*storage.Storage, error)
```

Comment:
typically the names of types that are functions will have `Func` at the end to help distinguish this